ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -121,3 +121,29 @@ executable db-analyser
                        -threaded
                        -rtsopts
                        "-with-rtsopts=-T -I0 -N2 -A16m"
+
+executable db-checker
```

Comment:
We have quite some DB's, if this only checks the LMDB maybe we want to give a more specific name, such as ledger-db-backends-check?